target/linux/generic/pending-5.15/101-Use-stddefs.h-instead-of-compiler.h.patch

```diff
@@ -1,22 +0,0 @@
-From d3c5b26768dbe990c4e1bd79e420c11ce7491d51 Mon Sep 17 00:00:00 2001
-From: OpenWrt community <openwrt-devel@lists.openwrt.org>
-Date: Wed, 13 Jul 2022 11:36:00 +0200
-Subject: [PATCH] swab: use stddefs.h instead of compiler.h
-
-Fix an issue with kernel headers that broke perf.
-
----
- include/uapi/linux/swab.h | 2 +-
- 1 file changed, 1 insertion(+), 1 deletion(-)
-
---- a/include/uapi/linux/swab.h
-+++ b/include/uapi/linux/swab.h
-@@ -3,7 +3,7 @@
- #define _UAPI_LINUX_SWAB_H
- 
- #include <linux/types.h>
--#include <linux/compiler.h>
-+#include <linux/stddef.h>
```

Comment:
This was added to stable here: https://github.com/gregkh/linux/commit/c160505c9b574b346031fdf2c649d19e7939ca11

Reply:
What action do you propose for `ramips/patches-5.15/100-PCI-mt7621-Add-MediaTek-MT7621-PCIe-host-controller-.patch`?  I believe I am not including the `+#include <linux/stddef.h>` you called out as getting added to stable.

Reply:
I think we should still remove `drivers/staging/mt7621-pci/pci-mt7621.c` in the `100-PCI-mt7621-Add-MediaTek-MT7621-PCIe-host-controller-.patch`, but we should also backport this patch https://github.com/torvalds/linux/commit/19098934f910b4d47cb30251dd39ffa57bef9523 as a separate patch for example in `target/linux/ramips/patches-5.15/107-...`

This patch is in upstream kernel, but it was backported to the old staging driver in kernel 5.15.

Reply:
Added and refreshed.  https://github.com/openwrt/openwrt/commit/31a8af0fd62ed2b0168c3674e45adeff291c3a51